embc-app/ClientApp/src/app/registration-maker/registration-maker.component.html

```diff
@@ -591,7 +591,7 @@ <h2 class="blue-header font-weight-bold">OTHER AGENCY REFERRALS MADE OUTSIDE OF
 
       <section class="space-between align-items-center mt-5 mb-4">
         <div>
-          <a routerLink="/dashboard">
+          <a routerLink="" (click)="cancel()">
```

Comment:
Did you test this link? (I have seen some of these with blank routerLink reload the whole page. In that case, use href="javascript:void(0)" instead.

Reply:
fixed